app/src/main/java/edu/uc/hornunjb/weatherwear/MainActivity.kt

```diff
@@ -25,11 +23,8 @@ class MainActivity : AppCompatActivity() {
         //Creating TextView Objects to bind to our fragment
         var temperatureText: TextView
         var cityText: TextView
-        var countryText: TextView
-        var humidityText: TextView
-        var pressureText: TextView
         var conditionsText: TextView
-        var windText: TextView
```

Comment:
removed the unused variables for simplicity. I believe unused code should not be in software deployments.